coordinator/gscoordinator/coordinator.py

```diff
@@ -819,6 +866,101 @@ def _fetch_gremlin_result(self, op: op_def_pb2.OpDef):
                 code=error_codes_pb2.OK, key=op.key, result=pickle.dumps(rlt)
             )
 
+    def _output(self, op: op_def_pb2.OpDef):
+        storage_options = pickle.loads(op.attr[types_pb2.STORAGE_OPTIONS].s)
```

Comment:
Passing the configuration using `json.loads/dumps` instead.

Reply:
done